.github/workflows/issue-pr-contrib-metrics.yaml

```diff
@@ -287,152 +287,177 @@ jobs:
         run: |
           set -euo pipefail
 
+          echo -e '# Contributions, Discussions, Advisories, and Issues' >> summary_report.md
```

Comment:
You probably could have saved yourself from a lot of typing instead of doing:

```bash
echo "blabla" >>summary_report.md
echo "ajwaj" >>summary_report.md
echo "more stuff" >>summary_report.md
```

did:

```bash
{
  echo "blabla"
  echo "ajwaj"
  echo "more stuff"
} >>summary_report.md

Reply:
Patches welcome 😉

Reply:
(Or even use { ... } | tee summary_report.md to get the markdown as debug output)